src/main/kotlin/org/wycliffeassociates/otter/jvm/app/ui/resourcetakes/view/TakesListView.kt

```diff
@@ -7,25 +7,26 @@ import javafx.scene.control.ListView
 import javafx.scene.layout.Priority
 import org.wycliffeassociates.otter.common.data.workbook.Take
 import org.wycliffeassociates.otter.common.device.IAudioPlayer
-import org.wycliffeassociates.otter.jvm.app.widgets.takecard.TakeEvent
+import org.wycliffeassociates.otter.jvm.app.widgets.takecard.events.PlayOrPauseEvent
 import org.wycliffeassociates.otter.jvm.app.widgets.takecard.resourcetakecard
 import tornadofx.*
 
-class TakesListView(items: ObservableList<Take>, audioPlayer: () -> IAudioPlayer) : ListView<Take>(items) {
-    private val lastTakeEvent: SimpleObjectProperty<TakeEvent?> = SimpleObjectProperty()
-
+class TakesListView(
+    items: ObservableList<Take>,
+    audioPlayer: () -> IAudioPlayer,
```

Comment:
why is this a lambda?

Reply:
I don't know- that was silly :)

Reply:
I remember now- TakesListView constructs TakeCards for each take in the list, and it needed to instantiate an AudioPlayer for each of the TakeCards. In reality this parameter should have been named something like `createAudioPlayer`. I just pushed a commit that removes this parameter as well as `lastPlayOrPauseEvent`, in favor of accepting a lambda function called `createTakeNode` that just constructs the card given a take object.

Here's another thought- I don't know if this entire class is really needed or not. I could just create a vanilla ListView instead in `RecordResourceFragment`, but perhaps this class still encapsulates the takes list view and reduces the code length of `RecordResourceFragment`... especially with the lengthy comment explaining the lack of caching the cells.